sold.h

```diff
@@ -115,6 +116,12 @@ class Sold {
                 }
             }
         }
+
+        // Even if the size of the TLS initialization image in the output
+        // shared object is zero, we emit a dummy image of the page size.
+        // Without this dummy image, memory segments of TLS and EHFrameHeader
+        // overlap.
+        s = std::max(s, LINUX_PAGE_SIZE);
```

Comment:
>  Assigning the same p_offset should be OK

Hmm... you are right. I carelessly thought this PR is right because it fixes all crashes in `OffsetFromAddr()`.